### PR TITLE
Make print statements conditional so that they will not show up in rspec

### PR DESCRIPTION
**Pull request by jhsu802701**
* The puts commands in the deployment tasks are disabled for the test environment only.
* The puts commands in the db/seeds.rb script are commented out to minimize excessive screen output when running the test suite.

### CODE REVIEW

File: db/seeds.rb [hunk 1]
```diff
@@ -77,21 +77,21 @@ def get_seed_specification
...
     end
     seed
   end
 
   def report_object_counts
-    puts "\nRecords written to the DB:\n\nCount  Class Name\n-----  ----------\n\n"
+    # puts "\nRecords written to the DB:\n\nCount  Class Name\n-----  ----------\n\n"
     active_record_classes.each do |klass|
-      puts "%5d  %s" % [klass.count, klass.name]
+      # puts "%5d  %s" % [klass.count, klass.name]
```
Comment (littleforest) [inline db/seeds.rb:94]: The `puts` statements are useful when first setting up the application or when re-seeding. Please revert these.

**Review by littleforest — CHANGES_REQUESTED** [review]

File: db/seeds.rb (hunk 1 above)
Comment (jhsu802701) [reply db/seeds.rb]: OK, I reinstated the puts commands.

**Comment by compwron** [comment]
I am ambivalent about this. @jhsu802701 can you describe more the benefit of this change? 

If you're looking for high priority work to do, check out the todo column on our issues board :) https://github.com/rubyforgood/casa/projects/1

If you're having trouble getting started, happy to do some interactive debugging in slack
And/or join office hours weds evening (Pacific time) - zoom link in slack
https://rubyforgood.herokuapp.com/ #casa channel

**Comment by littleforest** [comment]
I don't mind merging this as it stands now, but I don't think folks will regularly be adding this test environment exception for future after party tasks are going to be created. So it just seems that the code is going to end up kind of inconsistent.

**Comment by jhsu802701** [comment]
I'm trying to clean up the screen output from running the test suite.  So I disabled the puts commands for the lib/tasks functions in the test environment only.  (The puts commands still work in the development and production environments.)

**Review by compwron — APPROVED** [review]

**Comment by compwron** [comment]
You are invited to the CASA asynchronous retrospective-and-futurespective! Please take a look at https://bit.ly/casaretro (no login required) and add at least one note! Even if it's just "I was here and you are cool" or "omg y'all why do you keep messaging me" we want to hear it. <3
Next week Wednesday 18 November @ 6pm Pacific time we will Zoom about the board and talk about themes and takeaways. (Zoom link in our calendar at http://bit.ly/casacal or slack #casa channel https://rubyforgood.herokuapp.com/ )
Please ESPECIALLY add lots of notes if you won't be able to make it on Wednesday to talk!
* Note: the board will be monitored for spam; message me or any team lead if you see a problem
https://bit.ly/casaretro

**Pull request merged**